spec/lib/tasks/flag_rake_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+# see http://robots.thoughtbot.com/test-rake-tasks-like-a-boss
+describe 'flag_reservations:flag_overdue' do
+  include_context 'rake'
+
+  before(:all) { FactoryGirl.create(:app_config) }
+
+  it 'flags reservations due yesterday as overdue' do
+    overdue = FactoryGirl.build(:overdue_reservation, overdue: false)
+    not_overdue = FactoryGirl.build(:valid_reservation)
+    expect(!overdue.overdue)
+    expect(!not_overdue.overdue)
+    subject.invoke
+    expect(overdue.overdue)
```

Comment:
How is this passing? Since you're building reservations, not creating them, why is it affected by the rake task?